### PR TITLE
Mavlink bridge

### PR DESCRIPTION
**Pull request by denniszollo**
@njoubert @fnoble

### CODE REVIEW

**Comment by denniszollo** [comment]
@mookerji I incorporated your feedback on Niels original PR.  If this is fine by you, I'd like to merge and tag piksi_tools.  (the tag is useful for documentation)

**Comment by njoubert** [comment]
I'm a fan :)


-Niels

File: piksi_tools/mavlinkbridge.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                      help="specify the serial port to use.")
+  parser.add_argument("-b", "--baud",
+                      default=[DEFAULT_SERIAL_BAUD], nargs=1,
+                      help="specify the baud rate to use.")
+  parser.add_argument("-a", "--address",
+                      default=[DEFAULT_UDP_ADDRESS], nargs=1,
+                      help="specify the serial port to use.")
+  parser.add_argument("-p", "--udp-port",
+                      default=[DEFAULT_UDP_PORT], nargs=1,
+                      help="specify the baud rate to use.")
```
Comment (cbeighley) [inline piksi_tools/mavlinkbridge.py]: @denniszollo help string here looks incorrect

Comment (mookerji) [inline piksi_tools/mavlinkbridge.py]: +1

**Comment by mookerji** [comment]
Update help and merge whenever.

**Comment by denniszollo** [comment]
superceded by #74

**Pull request closed without merging**